apps/judicial-system/backend/migrations/20211117185851-update-case.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) =>
+      queryInterface.addColumn(
+        'case',
+        'initial_ruling_date',
+        {
+          type: Sequelize.DATE,
+          allowNull: true,
+        },
+        { transaction: t },
+      ),
+    )
```

Comment:
It would be nice if you could add a then clause after add column to update relevant existing cases, i.e. set initial ruling date for cases that have a parent.